src/net.h

```diff
@@ -188,6 +180,31 @@ enum class ConnectionType {
      * AddrMan is empty.
      */
     ADDR_FETCH,
+
+    /**
+     * Manual-block-relay connections are manually added nodes (via -addnode)
+     * that behave like BLOCK_RELAY connections in that they don't participate in
+     * transaction and address relay, and also share the properties of MANUAL
+     * connections. When connected to a privacy network such as
+     * Tor, manual block-relay-only connections to IPv4 or IPv6 nodes can be
+     * used for hardening against eclipse attacks, while at the same time
+     * making fingerprinting attacks more difficult and preserving a higher degree
+     * of privacy by not transmitting transactions and addresses over these links.
+     */
+    MANUAL_BLOCK_RELAY,
+};
+
+struct AddedNodeInfo {
+    std::string strAddedNode;
+    CService resolvedAddress;
+    bool fConnected;
+    bool fInbound;
+    ConnectionType conn_type;
+};
+
+struct AddedNodeEntry {
```

Comment:
I know that this is maybe a bit of a stretch for this PR, but I really don't like the addnode/addednode terminology, and would much prefer this to be named something like `ManualConnectionEntry`, which much more precisely describes the nature of these connections.

Reply:
The downside of `ManualConnectionEntry` is that it is not a connection, just a name of a node we'll try to connect to (which may succeed or not). I didn't really like the name AddedNodeEntry either though, but couldn't think of anything better. Maybe `ManualNodeEntry`?

Reply:
Or maybe `ManualConnectionTarget` or `DesiredManualConnection`? I'm not sure they're great either.